.github/workflows/docker_build_pr.yml

```diff
@@ -0,0 +1,31 @@
+name: Build Docker PR image
+
+# This one is disabled by default and can be enabled/customized based on specific needs:
+# create a branch, customize this for your branch (i.e. on.push.branches member) and push
+
+# remember: our valhalla fork will have
+
+on:
+  workflow_dispatch:
+
+jobs:
+  build_image:
+    name: Manual docker image build
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        submodules: 'recursive'
+        fetch-depth: 0
+
+    - name: Log in to GitHub Docker Registry
+      uses: docker/login-action@v2
+      with:
+        registry: ghcr.io
+        username: ${{ github.actor }}
+        password: ${{ secrets.GITHUB_TOKEN }}
+
+    - name: Build latest image
+      run: |
+        docker build -t ghcr.io/valhalla/valhalla:manually_triggered_build .
```

Comment:
The image name will be `manually_triggered_build` and will naturally be overwritten by the next build. Personally I don't think the manual build will be used much, so that shouldn't negatively interfere IMO. Quite opposite actually, it cleans up after itself. If we'd name it after the branch it's originating from I'd fear we wouldn't be very diligent deleting stuff when unused. 

I'll likely use it now and then for client previews (e.g. the recent GTFS one, now the traffic matrix), but that's like once a month.